panel/src/main/scala/hmda/panel/PanelCsvParser.scala

```diff
@@ -45,11 +45,15 @@ object PanelCsvParser extends InstitutionComponent with DbConfiguration {
 
     val timeout = 5.seconds
 
+    println("Cleaning DB...")
+    Await.result(repository.dropSchema(), timeout)
```

Comment:
Can we do the same without using `Await.result`, which is a blocking operation?. Not terribly concerned with this since this is its own application that will just run once